nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -131,38 +133,128 @@ def set_weight(
 
         del const_node
 
+    def insert_adapters(self, wc_params: WeightCompressionParameters, lora_A: Tensor, lora_B: Tensor, int8_lora: bool):
+        input_node = self.name_to_node_mapping[wc_params.node_with_weight.node_name].input_value(0)
+        activation_dtype = input_node.get_element_type()
+        should_add_convert_node = activation_dtype != ov.Type.f16
+        mm_node = self.name_to_node_mapping[wc_params.node_with_weight.node_name]
+
+        if int8_lora:
+            a_wc_params = deepcopy(wc_params)
+            a_wc_params.compression_config = WeightCompressionConfig()
+
+            A_W, _ = self._create_compression_subgraph(
+                weight=Tensor(lora_A),
+                wc_params=a_wc_params,
+                const_node_name=a_wc_params.node_with_weight.node_name + "_lora_A",
+                const_dtype=activation_dtype,
+                should_add_convert_node=should_add_convert_node,
+            )
+            B_W, _ = self._create_compression_subgraph(
+                weight=Tensor(lora_B),
+                wc_params=a_wc_params,
+                const_node_name=a_wc_params.node_with_weight.node_name + "_lora_B",
+                const_dtype=activation_dtype,
+                should_add_convert_node=should_add_convert_node,
+            )
+        else:
+            A_W = opset.constant(lora_A.data)
+            B_W = opset.constant(lora_B.data)
+
+        A_MM = opset.matmul(input_node, A_W, transpose_a=False, transpose_b=True)
```

Comment:
The `_create_compression_subgraph` uses `weight_port_id` from `a_wc_params`, but weight_port_id for lora adapters always is 1.